src/shared/securebits-util.c

```diff
@@ -6,30 +6,70 @@
 #include "extract-word.h"
 #include "securebits-util.h"
 #include "string-util.h"
+#include "strv.h"
 
-int secure_bits_to_string_alloc(int i, char **s) {
-        _cleanup_free_ char *str = NULL;
-        size_t len;
+/* match a single bit */
```

Comment:
We usually put documentation for functions inside the function at the start and not outside